test/test-lint-parse.cpp

```diff
@@ -85,7 +85,10 @@ TEST(test_lint,
   p.parse_and_visit_module(l);
   l.visit_end_of_module();
 
-  EXPECT_THAT(v.errors, IsEmpty());
```

Comment:
I think we should change `const` to `let` on line 80. The test's name tells me that `let` was intended anyway.